Formula/goplus.rb

```diff
@@ -19,7 +19,7 @@ class Goplus < Formula
 
   def install
     ENV["GOPROOT_FINAL"] = libexec
-    system "go", "run", "cmd/make.go", "--install"
+    system "go", "run", "cmd/make.go", "--build"
```

Comment:
What's the difference?

Reply:
When install with "brew install goplus --build-from-source"，"--install" will create a link to $HOME/gop but "--build" won't.

Reply:
It sounds like this isn't necessary then. Builds running inside `brew` cannot access the user's home directory by reading `$HOME`. On macOS, even if there was a way for the build to find out the user's home directory, the build cannot write to it due to the sandbox.

Reply:
OK. Thanks for review.